openslides_backend/action/actions/mediafile/upload.py

```diff
@@ -87,13 +88,23 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         instance["create_timestamp"] = round(time())
         filename_ = instance.get("filename", "")
         file_ = instance.pop("file")
-        instance["mimetype"] = mimetypes.guess_type(filename_)[0]
-        if instance["mimetype"] is None:
-            raise ActionException(f"Cannot guess mimetype for {filename_}.")
         decoded_file = base64.b64decode(file_)
+        mimetype_ = magic.from_buffer(decoded_file, mime=True)
+        possible_extensions = mimetypes.guess_all_extensions(mimetype_)
+        if mimetype_ is None:
+            raise ActionException(f"Cannot guess mimetype for {filename_}.")
+        matching_file_extension = False
```

Comment:
You should use the any-function for lines 96-103:

if not any([filename_.endswith(extension) for extension in possible_extensions]):
     raise ActionException ...